gustaf/io/meshio.py

```diff
@@ -72,18 +72,102 @@ def load(fname):
     return mesh
 
 
-def export(mesh, fname):
-    """Currently not implemented function.
+def export(mesh, fname, submeshes=None, **kwargs):
+    """Export mesh elements and vertex data into meshio and use its write
+    function. The definition of submeshes with identical vertex coordinates
+    is possible. In that case vertex numbering and data from the main mesh
+    are used. For more export options, refer to meshio's documentation
+    [https://github.com/nschloe/meshio](meshio)
+
+    .. code-block:: python
+
+        import gustaf
+        # define coordinates
+        v = np.array(
+            [
+                [0.0, 0.0, 0.0],
+                [1.0, 0.0, 0.0],
+                [0.0, 1.0, 0.0],
+                [1.0, 1.0, 0.0],
+                [0.0, 0.0, 1.0],
+                [1.0, 0.0, 1.0],
+                [0.0, 1.0, 1.0],
+                [1.0, 1.0, 1.0],
+            ]
+        )
+        # define triangle connectivity
+        tf = np.array(
+            [
+                [1, 0, 2],
+                [0, 1, 5],
+                [3, 2, 6],
+                [2, 0, 4],
+                [4, 5, 7],
+                [2, 3, 1],
+                [7, 5, 1],
+                [6, 7, 3],
+                [4, 6, 2],
+                [7, 6, 4],
+            ]
+        )
+        # init tri faces
+        mesh = gus.Faces(
+            vertices=v,
+            faces=tf,
+        )
+        gustaf.io.meshio.export(mesh, 'tri-mesh.stl')
 
     Parameters
     ------------
-    mesh: MESH_TYPES
-      Mesh to be exported.
+    mesh: Edges, Faces or Volumes
+      Input mesh
     fname: Union[str, pathlib.Path]
       File to save the mesh in.
+    submeshes: Iterable
+      Submeshes where the vertices are identical to the main mesh. The element
+      type can be identical to mesh.elements or lower-dimensional (e.g.
+      boundary elements).
```

Comment:
```suggestion
      boundary elements).
    **kwargs : arbitrary
      Any additional argument will be passed to the respective meshio `write`
      function. See meshio docs for more information
```

Reply:
The official Python type for arbitrary is `Any`.